bigc/api_client.py

```diff
@@ -66,35 +66,33 @@ def _get_standard_request_headers(self) -> dict:
 
     @staticmethod
     def _handle_error(response: requests.Response) -> NoReturn:
-        # TODO: Try to extract an error message from the response body
-
```

Comment:
Technically this is still a TODO. I believe v2 and v3 tend to have different error messages, so body parsing would need to happen at the API client level (here).

Reply:
https://github.com/MedShift/bigc/pull/35/commits/14a43d99af17f5d9fe66ec5571db4ba4c45241be